megamek/src/megamek/server/Server.java

```diff
@@ -7389,6 +7389,15 @@ private void processMovement(Entity entity, MovePath md,
                             }
                             // check for destruction
                             if (a.getSI() == 0) {
+                                // Lets auto-eject if we can!
+                                if (a.isAutoEject()
+                                    && (!game.getOptions().booleanOption(
+                                        OptionsConstants.RPG_CONDITIONAL_EJECTION) || (game
+                                                                            .getOptions().booleanOption(
+                                                OptionsConstants.RPG_CONDITIONAL_EJECTION) && a
+                                                                            .isCondEjectSIDest()))) {
```

Comment:
The wrapping on these lines is odd and makes it harder to read.

Reply:
Fixed